packages/react-devtools-shared/src/backend/views/Highlighter/Overlay.js

```diff
@@ -226,9 +227,24 @@ export default class Overlay {
 
     if (!name) {
       name = elements[0].nodeName.toLowerCase();
-      const ownerName = getOwnerDisplayName(elements[0]);
-      if (ownerName) {
-        name += ' (in ' + ownerName + ')';
+
+      const node = elements[0];
+      const hook: DevToolsHook =
+        node.ownerDocument.defaultView.__REACT_DEVTOOLS_GLOBAL_HOOK__;
+      if (hook != null && hook.rendererInterfaces != null) {
+        let ownerName = null;
+        // eslint-disable-next-line no-for-of-loops/no-for-of-loops
+        for (const rendererInterface of hook.rendererInterfaces.values()) {
+          const id = rendererInterface.getFiberIDForNative(node, true);
+          if (id !== null) {
```

Comment:
@bvaughn Just curious why this is a `!== null` instead of `!=null`? Are they using differently in the React codebase?

Reply:
`getFiberIDForNative` always returns either an id (number) or null.

Reply:
That's correct.

Reply:
Thank you so much for this @bvaughn! 😀